dvs10_Options.m

```diff
@@ -0,0 +1,883 @@
+% start trial with blank screen = baseline/pre-fixation epoch
+set_bgcolor(bg_trial)
+eventmarker(ec_iti_start)
+idle(i ti)
+eventmarker(ec_iti_end)
+
+%% 1. FIXATION
+% need to fixate centre (fixation breaks allowed). if no fixation acquired
+% trial is stopped, error screen and timeout
+run_scene(fixation_scene,ec_fix_screen_shown); 
+
+    % no fixation
+    if ~fixth.Success
+            trialerror(ec_out_nofix)
+            idle(nofix_timeout,bg_nofix)
+            eventmarker(ec_timeout_screenshown)
+    end
+    
+
+%% 2. OPTION 1 ONSET
+% option 1 is shown. no fixation required to continue. record if/which cues
+% are fixated.
+if fixth.Success
+TrialRecord.User.num_initiated_trials = TrialRecord.User.num_initiated_trials+1;
+
+option1_onset_time = run_scene(option1_scene,ec_option1_on);
+  
+    if mag_fth1.Success
+        eventmarker(ec_mag1_fix)
+        if x_position1_i==1     % option 1 on the left
+            if mag1_y_position_i==1     % mag is up
+                eventmarker(ec_upleft_fix)
+            else
+                eventmarker(ec_downleft_fix)
+            end
+        else
+            if mag1_y_position_i==1    
+                eventmarker(ec_upright_fix)
+            else
+                eventmarker(ec_downright_fix)
+            end
+        end
+
+        time_mag1_acquired = mag_fth1.AcquiredTime;                              
+
+        % log RT and precise timings
+        rt_mag1 = time_mag1_acquired - option1_onset_time;
+        dvs_rts_mag1 = [dvs_rts_mag1 rt_mag1];
+        dvs_ts_mag1_acquired = [dvs_ts_mag1_acquired time_mag1_acquired];
+        dvs_mag1_held = [dvs_mag1_held fta_mag1.FixTime];
+        bhv_variable('dvs_rts_mag1', dvs_rts_mag1);
+        bhv_variable('dvs_ts_mag1_acquired', dvs_ts_mag1_acquired);
+        bhv_variable('dvs_mag1_held',dvs_mag1_held)
+    end
+
+    if prob_fth1.Success
+        eventmarker(ec_prob1_fix)
+        if x_position1_i==1     % option 1 on the left
+            if mag1_y_position_i==2     % mag is down so prob is up
+                eventmarker(ec_upleft_fix)
+            else
+                eventmarker(ec_downleft_fix)
+            end
+        else
+            if mag1_y_position_i==2    
+                eventmarker(ec_upright_fix)
+            else
+                eventmarker(ec_downright_fix)
+            end
+        end
+
+        time_prob1_acquired = prob_fth1.AcquiredTime;
+
+        % log RT and precise timings
+        rt_prob1 = time_prob1_acquired - option1_onset_time;
+        dvs_rts_prob1 = [dvs_rts_prob1 rt_prob1];
+        dvs_ts_prob1_acquired = [dvs_ts_prob1_acquired time_prob1_acquired];
+        dvs_prob1_held = [dvs_prob1_held fta_prob1.FixTime];
+        bhv_variable('dvs_rts_prob1', dvs_rts_prob1);
+        bhv_variable('dvs_ts_prob1_acquired', dvs_ts_prob1_acquired);
+        bhv_variable('dvs_prob1_held',dvs_prob1_held)
+    end
+
+%% 3. OPTION 2 ONSET
+% option 2 is shown. no fixation required to continue.
+    idle(100)

```

Comment:
Why?

Reply:
But to be honest, he's used to it so don't change anything this late in the game